etc/kayobe/kolla-image-tags.yml

```diff
@@ -42,7 +42,9 @@ kolla_image_tags:
     ubuntu-jammy: 2024.1-ubuntu-jammy-20250627T102805
     ubuntu-noble: 2024.1-ubuntu-noble-20250627T102805
   ironic_dnsmasq:
+    rocky-9: 2024.1-rocky-9-20250716T041633
     ubuntu-jammy: 2024.1-ubuntu-jammy-20241218T141809
+    ubuntu-noble: 2024.1-ubuntu-noble-20250404T150323
```

Comment:
Can't we use the main `ironic` tag here?

Reply:
The ironic tag does not have dnsmasq unfortunately.